### PR TITLE
Kryptor: Add version 3.1.0

### LINKED ISSUE #7777: [Request] Add Kryptor
### Information

**Name:** 
Kryptor

**Description:** 
A simple, modern, and secure encryption and signing tool that aims to be a better version of age and Minisign.

**Homepage:** 
https://www.kryptor.co.uk/

**Download link(s):** 
https://github.com/samuel-lucas6/Kryptor/releases/download/v3.1.0/kryptor-windows.zip

**Some indication of popularity/repute:** 
Currently at 185 stars and over 4,000 downloads.

### PR DESCRIPTION
**Pull request by samuel-lucas6**
Closes #7777

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/kryptor.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://www.kryptor.co.uk/",
+    "description": "A simple, modern, and secure encryption and signing tool.",
```
Comment (rashil2000) [inline bucket/kryptor.json:3]:
```suggestion
    "description": "Open source file encryption and signing tool.",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
You publish signature/digest files with your releases. Use that in autoupdate hash extraction. See https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#adding-hash-to-autoupdate

File: bucket/kryptor.json (hunk 1 above)
Comment (samuel-lucas6) [reply bucket/kryptor.json:3]: Is this change necessary? I've used the same style as the [age](https://github.com/ScoopInstaller/Extras/commit/9287590c8efbb5a44cc43e5b5842d9a3a4872ed3) description.

**Comment by samuel-lucas6** [comment]
Not sure what I've done wrong @rashil2000.

File: bucket/kryptor.json (hunk 1 above)
Comment (rashil2000) [reply bucket/kryptor.json:3]: I took the description from your homepage itself. I feel it describes the software more succinctly:

image

File: bucket/kryptor.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "checkver": {
+        "github": "https://github.com/samuel-lucas6/Kryptor"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/samuel-lucas6/Kryptor/releases/download/v$version/kryptor-windows.zip",
+                "hash": {
+                    "url": "$url.DIGEST",
+                    "regex": "([a-fA-F0-9]{32,128})[\x20\t]+.*`$basename(?:[\x20\t]+\d+)?"
```
Comment (rashil2000) [inline bucket/kryptor.json:22]: Since this is JSON,  you need to escape the backslashes

File: bucket/kryptor.json (hunk 1 above)
Comment (samuel-lucas6) [reply bucket/kryptor.json:3]: Ok I'll go ahead and change it. Thanks for the explanation.

File: bucket/kryptor.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "checkver": {
+        "github": "https://github.com/samuel-lucas6/Kryptor"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/samuel-lucas6/Kryptor/releases/download/v$version/kryptor-windows.zip",
+                "hash": {
+                    "url": "$url.DIGEST",
+                    "regex": "([a-fA-F0-9]{32,128})[\\x20\t]+.*`$basename(?:[\\x20\t]+\\d+)?"
```
Comment (rashil2000) [inline bucket/kryptor.json:22]:
```suggestion
                    "url": "$url.DIGEST"
```

Actually, you don't need regex here at all

**Comment by samuel-lucas6** [comment]
Many thanks for the help @rashil2000!

**Pull request merged**